patches/0015-LPS-137763-If-contentsElement-is-defined-use-it-as-a.patch

```diff
@@ -0,0 +1,23 @@
+From 55190fea788bca0787230b5bd9e86a212fed8846 Mon Sep 17 00:00:00 2001
+From: Diego Nascimento <diego.nascimento@liferay.com>
+Date: Mon, 18 Oct 2021 18:46:04 -0300
+Subject: [PATCH] LPS-137763 If contentsElement is defined, use it as a
+ fallback for editor.ui.space('contents') (wysiwygarea)
+
+---
+ plugins/wysiwygarea/plugin.js | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/plugins/wysiwygarea/plugin.js b/plugins/wysiwygarea/plugin.js
+index 6c0a89d405..772839586c 100644
+--- a/plugins/wysiwygarea/plugin.js
++++ b/plugins/wysiwygarea/plugin.js
+@@ -42,7 +42,7 @@
+ 				iframe.setStyles( { width: '100%', height: '100%' } );
+ 				iframe.addClass( 'cke_wysiwyg_frame' ).addClass( 'cke_reset' );
+ 
+-				var contentSpace = editor.ui.space( 'contents' );
++				var contentSpace = editor.ui.space( 'contents' ) || editor.ui.contentsElement;
```

Comment:
There might be a bug here. What if both `editor.ui.space( 'contents' )` and `editor.ui.contentsElement` are set? Here, `editor.ui.space( 'contents' )` takes priority. But, in `themedui.js`, `editor.ui.contentsElement` takes priority:
```js
if (editor.ui.contentsElement) {
	editor.ui.contentsElement.setHtml('');
} else {
	editor.ui.space( 'contents' ).setHtml( '' );
}
```

Even if these two mutually exclusive, or duplicated, we should have consistent priorities for future-proofing.

Reply:
> we should have consistent priorities for future-proofing.

You're right. I changed themedui conditional to just use `editor.ui.contentsElement` in case if editor.ui.space is undefined ;)